collect_app/src/main/res/values/theme.xml

```diff
@@ -89,10 +89,15 @@
         </item>
 
         <!-- Material3 polyfill attributes (needed in Material Components yet). -->
-        <item name="materialButtonIconStyle">@style/Widget.Collect.Button.Icon</item>
-        <item name="materialButtonOutlinedIconStyle">@style/Widget.Collect.Button.OutlinedButton.Icon</item>
-        <item name="materialIconButtonOutlinedStyle">@style/Widget.Collect.Button.IconButton.Outlined</item>
-        <item name="borderlessButtonIconStyle">@style/Widget.Collect.Button.TextButton.Icon</item>
+        <item name="materialButtonIconStyle">@style/Collect.Button.Icon</item>
+        <item name="materialButtonOutlinedIconStyle">@style/Collect.Button.OutlinedButton.Icon</item>
+        <item name="materialIconButtonOutlinedStyle">@style/Collect.Button.IconButton.Outlined</item>
+        <item name="borderlessButtonIconStyle">@style/Collect.Button.TextButton.Icon</item>
+        <item name="materialButtonStyle">@style/Collect.Button</item>
+        <item name="materialButtonOutlinedStyle">@style/Collect.Button.OutlinedButton</item>
+        <item name="borderlessButtonStyle">@style/Collect.Button.TextButton</item>
+        <item name="materialAlertDialogTheme">@style/Theme.Collect.Dialog.Alert</item>
+        <item name="widgetMaterialButtonStyle">@style/Widget.Collect.Button</item>
```

Comment:
I really like the approach of using a theme attribute here. That would support the idea we've talked about of having a `widgets` module as we'd be able to declare and use the attribute there and have `collect_app` actually declare the styling.

Could we move this attribute lower in the file and add a comment that it's an attribute for question widgets? I think also naming it `questionWidgetButtonStyle` or `widgetButtonStyle` would make it clearer that it's not part of Material Components.